src/components/document/document-content.tsx

```diff
@@ -155,6 +156,10 @@ export class DocumentContentComponent extends BaseComponent<IProps, IState> {
     });
   }
 
+  private renderSpacer = () => {
+    return this.props.readOnly ? null : <div className="spacer" />;
```

Comment:
I chose not to add the spacer to read only documents because 1) it pushed the "open" button down on problem tabs and 2) it will never be necessary to expand a tile in a read-only view